.github/workflows/people.yml

```diff
@@ -58,26 +58,71 @@ jobs:
             exit 1
           fi
 
-  build-front-desk:
```

Comment:
We potentially don't need the job `build-front-desk` anymore, because the job `test-e2e` build the app, so test the build process (lint / build).
But we could keep it for better visibility, up to you.